unlock-app/src/__tests__/actions/error.test.js

```diff
@@ -1,15 +1,36 @@
-import React from 'react'
-import { setError, SET_ERROR } from '../../actions/error'
+import {
+  setError,
+  SET_ERROR,
+  RESET_ERROR,
+  resetError,
+} from '../../actions/error'
 
 describe('error actions', () => {
   it('should create an action to set the error', () => {
-    const error = {
```

Comment:
Same comment as in other PR. What is the purpose of this in the context of this PR? 
This will create merge conflicts for you...